layout-management-services/src/main/java/org/exoplatform/layoutmanagement/rest/EntityBuilder.java

```diff
@@ -131,4 +133,19 @@ public static List<SiteRestEntity> toSiteRestEntities(List<Site> sites) {
   private static OrganizationService getOrganizationService() {
     return CommonsUtils.getService(OrganizationService.class);
   }
+
+  private static Map<String, Object> computePermission(String permission) {
```

Comment:
```suggestion
  private static Map<String, Object> computeEditPermission(String editPermission) {
```

Reply:
it is a method to compute one permission which is used in other methods  "computePermissions"